src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,30 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, setActiveTabId }) => {
```

Comment:
<img width="868" alt="image" src="https://github.com/mate-academy/react_tabs-js/assets/70220667/85ee4fb4-2adf-4d94-89dd-41d2ed5131a1">
you should pass these props